src/VisualStudio/Core/Test.Next/Services/AssetProviderTests.cs

```diff
@@ -41,7 +41,7 @@ public async Task TestVisualBasicParseOptionsSynchronization()
 
         private static async Task TestAssetAsync(object data)
         {
-            var sessionId = 0;
+            var sessionId = Checksum.Create(ImmutableArray.CreateRange(Guid.NewGuid().ToByteArray()));
```

Comment:
any dummy value woudl do.  The mocks in these code actually end up not even using this.